problemtools/verifyproblem.py

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env python2
 # -*- coding: utf-8 -*-
+from __future__ import print_function
```

Comment:
Could you remove the python3 print stuff from the PR?  (Not saying it is a bad idea, but better then to make a separate PR which applies this change to all of problemtools, not just to verifyproblem.py.)

Reply:
Why the opposition to drive-by cleanups :( But sure, I can split that out...

Reply:
In this case it was mostly an opposition to drive-by cleanups that leave things inconsistent (with some programs using Python 2 prints and others using Python 3 prints). (In other cases it has been a case of "let's extract the parts that can be merged immediately to reduce rot.")